### PR TITLE
Elemental analysis missing documentation

### LINKED ISSUE #26237: Missing documentation for Muon Elemental Analysis interface
The new elemental analysis interface does not have a documentation page.

### PR DESCRIPTION
**Pull request by BassoLorenzo**
Added documentation for Elemental Analysis interface.

**To test:**

- Build the documentation using the target `docs-html`
- Open `Interfaces->Muon Elemental Analysis`

Fixes #26237.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/interfaces/Muon Elemental Analysis.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+Plotting
+--------
+Tick the boxes corresponding to the detector you want to display. For each detector, a separate plot should appear in
+the same window. Each plot contains 3 lines: *Prompt*, *Delayed* and *Total*.
+
+.. image:: ../images/MuonElementalAnalysisPlot.png
+  :align: center
+
+To remove one or more lines on the top right corner of the plot press *Remove line*, a table should appear. Select the
```
Comment (LolloB) [inline docs/source/interfaces/Muon Elemental Analysis.rst]:
```suggestion
To remove one or more lines on the top right corner of the plot press *Remove line*, a table will appear. Select the
```

File: docs/source/interfaces/Muon Elemental Analysis.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  :align: center
+
+Loading
+-------
+Enter a run number (e.g. 6295) and press *Load*. If the number entered does not correspond to any available run,
+the program will not load any file, nor return an error.
+
+Plotting
+--------
+Tick the boxes corresponding to the detector you want to display. For each detector, a separate plot should appear in
```
Comment (LolloB) [inline docs/source/interfaces/Muon Elemental Analysis.rst]:
```suggestion
Tick the boxes corresponding to the detector you want to display. For each detector, a separate plot will appear in
```

File: docs/source/interfaces/Muon Elemental Analysis.rst [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+To remove one or more lines on the top right corner of the plot press *Remove line*, a table should appear. Select the
+lines you want to remove and press *OK*.
+Similarly, to remove one plot press *Remove plot*, select the plot you wish to remove and press *OK*.
+
+Displaying peaks
+----------------
+Elements whose name appears in grey lack any peak data and thus cannot be plotted.
+To observe the peaks of any other, click on its label which should turn yellow.
+Now plot data from one of the detectors and select all the peaks you would like to examine: main, secondary,
+gamma and electronic. Lines should appear for each of the peaks.
```
Comment (LolloB) [inline docs/source/interfaces/Muon Elemental Analysis.rst]:
```suggestion
gamma and electronic. Lines will appear for each of the peaks.
```

File: docs/source/interfaces/Muon Elemental Analysis.rst [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+Lines corresponding to different elements should appear in different colours.
```
Comment (LolloB) [inline docs/source/interfaces/Muon Elemental Analysis.rst]:
```suggestion
Lines corresponding to different elements will appear in different colours.
```

File: docs/source/interfaces/Muon Elemental Analysis.rst [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  :align: center
+
+To remove one or more lines on the top right corner of the plot press *Remove line*, a table will appear. Select the
+lines you want to remove and press *OK*.
+Similarly, to remove one plot press *Remove plot*, select the plot you wish to remove and press *OK*.
+
+Displaying peaks
+----------------
+Elements whose name appears in grey lack any peak data and thus cannot be plotted.
+To observe the peaks of any other, click on its label which should turn yellow.
```
Comment (LolloB) [inline docs/source/interfaces/Muon Elemental Analysis.rst]:
```suggestion
To observe the peaks of any other, click on its label which will turn yellow.
```

**Review by NickDraper — DISMISSED** [review]
I'd like to see a screenshot added and a list of things that the new GUI does better than the old one (aimed at the users)

**Review by Matthew-Andrew — APPROVED** [review]
Documentation builds without error, is accessible and makes sense.

**Pull request merged**